fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -930,7 +931,10 @@ public static MaterializedViewRewriteMode parse(String str) {
     private boolean enableAsyncProfile = true;
 
     @VariableMgr.VarAttr(name = BIG_QUERY_PROFILE_SECOND_THRESHOLD)
-    private int bigQueryProfileSecondThreshold = 0;
+    private long bigQueryProfileSecondThreshold = 0;
+
+    @VariableMgr.VarAttr(name = BIG_QUERY_PROFILE_MILLISECOND_THRESHOLD, flag = VariableMgr.INVISIBLE)
```

Comment:
1. The `BIG_QUERY_PROFILE_MILLISECOND_THRESHOLD` can replace `BIG_QUERY_PROFILE_SECOND_THRESHOLD`?
2. If you think the unit of `BIG_QUERY_PROFILE_SECOND_THRESHOLD` is too small, maybe `BIG_QUERY_PROFILE_THRESHOLD=10m/10s/10ms`, support time unit is a better name?

Reply:
Yes, only keep `BIG_QUERY_PROFILE_MILLISECOND_THRESHOLD` or provide a unit variable is a better choice. But If we do so, there may be a compatibility problem, that's why I keep them both.

Reply:
I think it over again, you're right, offer another unit variable is a better choice and don't have compatible problem.